packages/preview/splash/0.4.0/src/palettes/xkcd.typ

```diff
@@ -0,0 +1,951 @@
+#let xkcd_rgb = (
```

Comment:
You may want to rename this palette to `xkcd-rgb` to be more consistent with the rest of the package.